packages/veritone-react-common/src/components/FormBuilder/FormItems/CheckBoxes.js

```diff
@@ -19,6 +19,8 @@ export default function Checkboxes({
   items,
   error
 }) {
+  const styles = useStyles({});
```

Comment:
Remove if not used

Reply:
Actually we are using it, so we can not remove it.